models/user.go

```diff
@@ -130,9 +131,15 @@ func NewUserUpdate(user *vmod.User) *UserUpdate {
 	}
 }
 
-func UserPipeline() (pipe *vmdb.Pipeline) {
+func UserPipeline(user bool) (pipe *vmdb.Pipeline) {
 	pipe = vmdb.NewPipeline()
-	pipe.LookupUnwind(AddressesCollection, "_id", "user_id", "address")
+	if user == true {
+		pipe.LookupUnwind(AddressesCollection, "_id", "user_id", "address")
+	} else {
+		// TODO: Get rid of the address in the resonse at this point
+		pipe.LookupUnwind(AddressesCollection, "_id", "user_id", "address")
+		pipe.Append(bson.D{{Key: "$addFields", Value: bson.D{{Key: "address_id", Value: "$address._id"}}}})
```

Comment:
```		
pipe.LookupUnwind(AddressesCollection, "_id", "user_id", "address_dummy")
pipe.Append(bson.D{{Key: "$addFields", Value: bson.D{{Key: "address_id", Value: "$address_dummy:._id"}}}})
```
Mach einfach aus dem `address` ein `address_dummy`. Wenn der Key in der go struct nicht als bson definiert ist, wird er auch nicht geladen.